text/0045-nft-deposits-asset-hub.md

```diff
@@ -0,0 +1,246 @@
+|---------------------------|:----------------------:|:-----------------------:|
+| `collectionDeposit`       | 0.1                    | 0.006688                |
+| `itemDeposit`             | 0.001                  | 0.000167                |
+| `metadataDepositBase`     | 0.006709666617         | 0.0006709666617         |
+| `attributeDepositBase`    | 0.00666666666          | 0.000666666666          |
+
+### Enhanced Approach to Further Lower Barriers for Entry
+
+This RFC proposes further lowering these deposits below the rate normally charged for such a storage
+footprint. This is based on the economic argument that sub-rate deposits are a subsididy for growth
+and adoption of a specific technology. If the NFT functionality on Polkadot gains adoption, it makes
+it more attractive for future entrants, who would be willing to pay the non-subsidized rate because
+of the existing community.
+
+**Proposed Rate Adjustments**
+
+```rust
+parameter_types! {
+	pub const NftsCollectionDeposit: Balance = system_para_deposit(1, 130);
+	pub const NftsItemDeposit: Balance = system_para_deposit(1, 164) / 40;
+	pub const NftsMetadataDepositBase: Balance = system_para_deposit(1, 129) / 10;
+	pub const NftsAttributeDepositBase: Balance = system_para_deposit(1, 0) / 10;
+	pub const NftsDepositPerByte: Balance = system_para_deposit(0, 1);
+}
+```
+
+This adjustment would result in the following DOT and KSM deposit values:
+
+| **Name**                  | **Proposed Rate Polkadot** | **Proposed Rate Kusama** |
+|---------------------------|:--------------------------:|:------------------------:|
+| `collectionDeposit`       | 0.20064 DOT                | 0.006688 KSM             |
+| `itemDeposit`             | 0.005 DOT                  | 0.000167 KSM             |
+| `metadataDepositBase`     | 0.002 DOT                  | 0.0006709666617 KSM      |
+| `attributeDepositBase`    | 0.002 DOT                  | 0.000666666666 KSM       |
+
+### Short- and Long-Term Plans
+
+The plan presented above is recommended as an immediate step to make Polkadot a more attractive
+place to launch NFTs, although one would note that a forty fold reduction in the Item Deposit is
+just as arbitrary as the value it was replacing. As explained earlier, this is meant as a subsidy to
+gain more momentum for NFTs on Polkadot.
+
+In the long term, an implementation should account for what should happen to the deposit rates
+assuming that the subsidy is successful and attracts a lot of deployments. Many options are
+discussed in the [Addendum](#addendum).
+
+The deposit should be calculated as a function of the number of existing collections with maximum
+DOT and stablecoin values limiting the amount. With asset rates available via the Asset Conversion
+pallet, the system could take the lower value required. A sigmoid curve would make sense for this
+application to avoid sudden rate changes, as in:
+
+$$ \frac{\mathrm{min(DOT deposit, stable deposit)} }{\mathrm{1 + e^{a - b * x}} }$$
```

Comment:
Yea LGTM. Made some [script](https://github.com/ggwpez/substrate-scripts/blob/a0799650caaf478d7bf1eaf8961f561cf070c1a1/nft-deposit-curve.py) to inspect the params.    
<img src="https://raw.githubusercontent.com/ggwpez/substrate-scripts/master/nft-deposity.png"></img>

Reply:
@ggwpez I did a PR ([link](https://github.com/ggwpez/substrate-scripts/pull/4)) to your code so that the function is a bit adjusted to reflect the changes in the function below.

https://github.com/polkadot-fellows/RFCs/assets/30662672/d53f4b7c-f011-4ae2-84e1-e63f20550891

Reply:
Perfect, yea we can use this to find good params for making it auto-adjust. Thanks!